helpers/auth/authentication.py

```diff
@@ -118,13 +123,15 @@ def wrapper(*args, **kwargs):
                         status = 401
                         handle_http_error(message, status, expected_args)
 
-                    if response['values'][0]['location']:
-                        user.location = \
-                            response['values'][0]['location']['name']
-                        user.save()
-                    else:
-                        user.location = "Nairobi"
-                        user.save()
+                    for value in response['values']:
+                        if user.email == value["email"]:
+                            if value['location']:
+                                user.location = \
```

Comment:
Avoid using `\` here, it compromises the readability of the code

Reply:
@Bonifase Done.